src/components/layouts/aboutLayout.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Head from "next/head";
+import Header from "../general/Header";
+import Footer from "../general/Footer";
+import { Container } from "@material-ui/core";
+import { makeStyles, ThemeProvider } from "@material-ui/core/styles";
+import theme from "../../themes/aboutTheme";
+
+const useStyles = makeStyles({
+  mainHeading: {
+    textAlign: "center",
+    marginTop: -16
+  },
+  main: {
+    padding: 0
+  }
+});
+
+export default function Layout({ children }) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Head>
+        <link
+          rel="stylesheet"
+          href="https://use.fontawesome.com/releases/v5.11.2/css/all.css"
```

Comment:
I considered it. The main reason why I kept the fontawesome icons was because I didn't know how to dynamically import the icons that the info.json and links.json files tell me to import.

Reply:
Personally, I think it'd be best to use only one set of icons. That will make things less confusing for us, and help us standardize on one thing visually.

We could solve this in one of two ways:

1. Convert `info.json` to use JavaScript (`info.js` or similar). Then you could do something like this:

   ```jsx
   // ...
   {
      title: "Share your actions",
      icon: PlusIcon,
      // ...
   ```

2. Add something that maps an icon to its Material UI icon. This would take a string and convert it into the Material UI component. Something like:

   ```js
   const materialIconByName = {
      plus: PlusIcon,
      chart: ChartIcon,
      // ...
   };
   ```

Reply:
I will look to replace the FA icons with material ui native icons.

Reply:
I replaced them with Material UI icons using option 1